TrafficCapture/dockerSolution/src/main/docker/migrationConsole/workflows/README.md

```diff
@@ -0,0 +1,246 @@
+
+### Bonus Features
+
+After building out a workflow that the user is happy with, or in the middle of that run, a user may decide to test performance on additional clusters.  With a simple declarative configuration with minimal configurations, the system can stamp out a second pipeline to an additional target cluster.  The UX can/should keep the workflows in sync as changes are made to one.  Once the user is satisfied with a complete migration, they could extract the complete migration and load it onto another environment (pre-prod to prod, across customer partitions, etc).
+
+Notice that our tooling keeps this map around and uses the workflow state to manage the context, knowing when one stage is done, another may need to be rolled back, etc.  That frees us from needing to perform tedious bookkeeping.  Instead, we can focus on enhancing the workflow structure to pause until a workflow receives user-input or to allow workflow modifications (new stages/ modifications).
+
+### Idealized User Workflow
+
+Below is a simplified high-level workflow of the steps that would need to be executed for a full-migration.  Some of these steps may be omitted or substituted.  All of them could be restarted, though that may require rolling back to even earlier states.
+
+```mermaid
+flowchart
+    install_migration_tools --> start_capture --> create_snapshot --> metadata_index[Metadata] --> rfs[RFS] --> replay[Replay]
+    rfs --> ongoing_validation --> swing_proxy_to_target --> upgrade_clients --> disable_capture --> teardown_migration_tools
+    upgrade_clients --> deprovision_source
+    replay --> ongoing_validation
+```
+
+## Inner Workflows
+
+Some of the steps above will be composed of other workflows.  An invocation of RFS or the replayer involves deploying compute nodes, managing their scaling, and observing for an exit criteria.  For example: all documents have been migrated or the lag between the source and target is low and there are no 4xx errors.
+
+### Start Capture
+
+The start capture workflow is worth describing because it's an example of a simple model that helps to facilitate reasoning.  In this case "all proxies are healthy" sounds like an easy and obvious step.  However, how many proxies have been deployed?  How does the rest of the system know the global health?  We don't have that global state today, but with workflow modeling, we can zoom in on steps, let them express more granular conditions, and test them for correctness! 
+
+```mermaid
+flowchart
+    initialize_kafka_cluster --> initialize_kafka_topic --> signal_proxy_to_write
+    rollout_capture_proxy_across_fleet --> all_proxies_are_healthy_and_not_capturing --> signal_proxy_to_write
+    signal_proxy_to_write --> all_proxies_are_capturing
+```
+
+## Example User-Driven Workflow (including A/B testing)
```

Comment:
I'm lost past this point, maybe trim this down to what is going in right now or I'm missing out on how the workflow management systems operates.

Reply:
Is that happening in this PR or future PR?  I'd prefer less, more trusted content in our readme.md otherwise its easy to get stale and confusing for our future selves.